packages/common/src/authorization/projectMemberAbility.ts

```diff
@@ -19,6 +19,12 @@ export const projectMemberAbilities: Record<
         });
         can('view', 'Space', {
             projectUuid: member.projectUuid,
+            isPrivate: false,
+        });
+        can('view', 'Space', {
+            projectUuid: member.projectUuid,
+            isPrivate: true,
+            access: member.userUuid,
```

Comment:
Slightly confusing notation by casl: https://casl.js.org/v6/en/guide/conditions-in-depth#match-specified-item-in-an-array-property

```
# property equals 'hello'
property: 'hello'

# property is an array containing 'hello'
property: 'hello'
```